contributors/devel/sig-architecture/validation-suites.md

```diff
@@ -0,0 +1,92 @@
+# Validation Suite Testing in Kubernetes
+
+A Kubernetes Validation suite is a subset of e2e tests that are used to
+validate functionality of an optional capability of Kubernetes for use
+with a specific release. Validation suites can grow or shrink over time, and can also
```

Comment:
I think it should be clear that, like conformance tests, these are tests that are intended to be used outside of the standard CI, and run against an arbitrary cluster. They are used to validate that a given provider/distribution implements a feature in the standard way. The goal in my mind is that a validation suite should allow us to enforce that a given feature works in the same manner across distributions. If that's the goal (and it is left to, say, profiles), then we should also explicitly state that. But I am not sure what the point is if that's not the goal.

Reply:
Hi @johnbelamaric I think there is a lot of history on this topic you should catch up on. We covered this at the f2f meeting at KubeCon Seattle https://youtu.be/dueJApGGAo0
But essentially validation suites were not something that was mandatory for each distribution to run. We needed a way for both vendors and  the e2e test team to create groups of test based on functionality that was optional.  The conformance tests and possibly the creation of conformance  profiles is something that takes a large amount of discussion and consensus.  Validation suites are created and controlled by SIGs and can be experimental and can change.  Conformance tests are more official and tied to the conformance certification program.  @dims had good examples for why we should do validation suites first.

Reply:
Ok. Well, if you can address the other comments maybe we can get this merged soon?

Reply:
@johnbelamaric This was handled by adding clarity about what validation suites actually were used for and how they were different from conformance profiles. The portions where I discuss how validation suites can grow and shrink over time and their flexibility of a validation suite to change based on experimentation and feedback were the key characteristics that differentiated the validation suites from conformance profiles.  Between this and removing the section that @dims asked to be removed I believe I addressed  the concerns that @quinton-hoole had regarding the taxonomy.  So unless folks still don't understand how validation suites are different than conformance profiles due to the characteristics I describe, I believe this is ready to merge.

Reply:
@bradtopol Please change the line#1 to not say 'subset of e2e tests'

Reply:
Wiil do!